packages/app/src/server/service/config-loader.ts

```diff
@@ -268,6 +268,12 @@ const ENV_VAR_NAME_TO_CONFIG_INFO = {
     type:    ValueType.BOOLEAN,
     default: false,
   },
+  ELASTICSEARCH_REINDEX_ON_BOOT: {
+    ns:      'crowi',
+    key:     'app:elasticsearchReindexOnBoot',
+    type:    ValueType.BOOLEAN,
+    default: true,
```

Comment:
psl change it to `false` since we don't want reindexing to work unless we explicitly set it to true